api/pom.xml

```diff
@@ -110,6 +110,33 @@
 				<filtering>true</filtering>
 			</testResource>
 		</testResources>
+
+		<plugins>
+			<plugin>
+				<groupId>com.googlecode.maven-java-formatter-plugin</groupId>
```

Comment:
@k-joseph This is just the api/pom.xml. The top-level pom file has the same changes as the pom file you linked.